src/components/elements/icon/index.ts

```diff
@@ -0,0 +1,5 @@
+import { Alien, CaretDown, RocketLaunch, Sparkle, Book } from '@phosphor-icons/react/dist/ssr';
```

Comment:
It needs the `use client`, because the phosphor uses a context to handle global styles. Currently the build is failing because of this.
```suggestion
'use client';

import { Alien, CaretDown, RocketLaunch, Sparkle, Book } from '@phosphor-icons/react/dist/ssr';
```